charts/tekton-pipelines/templates/common/tasks/kustomize.yaml

```diff
@@ -67,7 +67,7 @@ spec:
           CONFIG=$(realpath .yamlfix.toml)
         fi
 
-        find apps -iname "*.yaml" -exec yamlfix --config-file=$CONFIG {} \;
+        find .kubernetes apps -iname "*.yaml" -exec yamlfix --config-file=$CONFIG {} \;
```

Comment:
I suggest another solution for it here: https://github.com/saritasa-nest/saritasa-devops-helm-charts/pull/59

Do not hardcode paths, since we already have parameter `$(params.kustomize_overlay_path)`